projects/cashmere/src/lib/app-switcher/app-switcher-application-link/app-switcher-links.component.ts

```diff
@@ -0,0 +1,11 @@
+import {Component, Inject} from '@angular/core';
+import {IAppSwitcherService, APP_SWITCHER_SERVICE} from '../app-switcher-interfaces';
+
+@Component({
+    selector: 'hc-app-switcher-links',
+    templateUrl: './app-switcher-links.component.html',
+    styleUrls: ['./app-switcher-links.component.scss']
+})
+export class AppSwitcherLinksComponent {
+    constructor(@Inject(APP_SWITCHER_SERVICE) public _appSwitcherService: IAppSwitcherService) {}
```

Comment:
This happens quite a few places in cashmere, I guess its confusing here because the default accessor is public, so this is superfluous. It is public to allow the template access, but has the underscore to not show up in the documentation.